src/main/jbake/content/using-git.ad

```diff
@@ -0,0 +1,178 @@
+* Click the "Create pull request" button and name the request "OPENNLP-XXXX" (uppercase). This will connect the
+comments of the PR to the mailing list and JIRA comments.
+* From now on the PR lives on GitHub's apache/opennlp. You can use the commenting UI there. 
+* If you are looking for a review or sharing with someone else say so in the comments but do not worry about 
+automated merging of your PR -- you will have to do that later. The PR is tied to your branch so you can respond to 
+comments, fix code, and commit from your local repository. They will appear on the PR page and be mirrored to JIRA 
+and to the mailing list. 
+
+When you are satisfied and want to push it to Apache's remote repository proceed with **Merging a PR**
+
+## How to create a PR (contributors)
+
+Create pull requests: <<anchor-1,[1]>>. 
+
+Pull requests are made to apache/opennlp repository on GitHub. In the GitHub UI you should pick the master 
+branch to target the PR as described for committers. This will be reviewed and commented on so the merge is 
+not automatic. This can be used for discussing a contributions in progress.
+
+## Merging a PR (yours or contributors) 
+
+Start with reading <<anchor-2,[2]>> (merging locally). 
+
+Remember that pull requests are equivalent to a remote GitHub branch with potentially a multitude of commits. 
+In this case it is recommended to squash remote commit history to have one commit per issue, rather 
+than merging in a multitude of contributor's commits. In order to do that, as well as close the PR at the 
+same time, it is recommended to use **squash commits**.
+
+Merging pull requests is equivalent to a "pull" of a contributor's branch:
+
+    git checkout master      # switch to local master branch
+    git pull apache master   # fast-forward to current remote HEAD
+    git pull --squash https://github.com/cuser/opennlp cbranch  # merge to master 
+
+`--squash` ensures all PR history is squashed into single commit, and allows committer to use his/her own
+message. Read git help for merge or pull for more information about `--squash` option. In this example we 
+assume that the contributor's GitHub handle is "cuser" and the PR branch name is "cbranch". 
+Next, resolve conflicts, if any, or ask the contributor to rebase on top of master, in case the PR went out of sync.
+
+If you are ready to merge your own (committer's) PR you probably only need to merge (not pull), since you have a local
+copy that you have been working on. This is the branch that you used to create the PR.
+
+    git checkout master      # switch to local master branch
+    git pull apache master   # fast-forward to current remote HEAD
+    git merge --squash opennlp-xxxx
+
+Remember to run regular patch checks, build with tests enabled, and change CHANGELOG.
+
+If everything is fine, you now can commit the squashed request along the lines
+
+    git commit --author <contributor_email> -a -m "OPENNLP-XXXX description closes #ZZ"
```

Comment:
Doing git rebase and merge is a simpler process because the name/email and commit message doesn't need to be copied by hand.